r5/variables.bal

```diff
@@ -0,0 +1,409 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// AUTO-GENERATED FILE.
+// This file is auto-generated by WSO2 Healthcare.
+
+# Terminology processor instance
+public final TerminologyProcessor terminologyProcessor = new ();
+
+# FHIR registry instance
+public final FHIRRegistry fhirRegistry = new ();
+
+# Modifier to Search parameter compatibility mapping
+final readonly & ModifierMap SEARCH_PARAM_MODIFIER_MAP = {
+    modifierTypeMapping: {
+        "above": [REFERENCE, TOKEN, URI],
+        "below": [REFERENCE, TOKEN, URI],
+        "code-text": [REFERENCE, TOKEN],
+        "contains": [STRING, URI],
+        "exact": [STRING],
+        "identifier": [REFERENCE],
+        "in": [TOKEN],
+        "missing": [DATE, NUMBER, QUANTITY, REFERENCE, STRING, TOKEN, URI],
+        "not": [TOKEN],
+        "not-in": [REFERENCE, TOKEN],
+        "of-type": [TOKEN],
+        "text": [REFERENCE, TOKEN, STRING],
+        "text-advanced": [REFERENCE, TOKEN]
+    }
+};
+
+# Search Parameters for all resources
+public final readonly & map<CommonSearchParameterDefinition> COMMON_SEARCH_PARAMETERS = {
```

Comment:
According to spec [doc](https://hl7.org/fhir/R5/searchparameter-registry.html) there are 2 more search parameters under `Resource`.

1. [_filter](https://hl7.org/fhir/R5/search_filter.html) - (Resource-filter | Filter search parameter which supports a more sophisticated grammar for searching.)
2. _query - ( Resource-query | A custom search profile that describes a specific defined query operation)